controllers/contentlibrary/clustercontentlibraryitem/clustercontentlibraryitem_controller.go

```diff
@@ -178,23 +170,22 @@ func (r *Reconciler) ReconcileNormal(ctx goctx.Context, cclItem *imgregv1a1.Clus
 			return err
 		}
 
-		syncErr = r.syncImageContent(ctx, cvmi, cclItem)
+		syncErr = r.syncImageContent(ctx, logger, cvmi, cclItem)
 		// Do not return syncErr here as we still want to patch the updated fields we get above.
 		return nil
 	})
 
 	logger = logger.WithValues("operationResult", opRes)
+	defer func() {
+		// createOrPatchErr can be updated below.
+		r.Metrics.RegisterVMIResourceResolve(logger, cvmi.Name, "", createOrPatchErr == nil)
```

Comment:
This seems somewhat odd to me to count: in the steady state it is basically just counting reconciles which I don't view as that interesting or useful. Maybe instead, only do it when `OpRes != None` so we actually did a Create/Patch/Update?

Reply:
In the steady state, this is just a no-op as we set the metrics value to either 1 or 0, indicating the success or failure of a CVMI reconciliation, rather than counting the number of reconciliations. While we could avoid the no-op with a `OpRes != None`, it wouldn't alter the metrics values.